src/components/Editor/Preview/Popup.js

```diff
@@ -179,6 +179,8 @@ export class Popup extends Component {
   getPreviewType(value: any) {
     if (
       typeof value == "boolean" ||
+      (typeof value == "string" && value.length < 10) ||
+      (typeof value == "number" && value.toString().length < 10) ||
```

Comment:
what i think is happening here is that we will avoid showing a popup when you hover on a number, but we will show it when you hover on a variable?!?! is that correct?

Reply:
Yup... the getPreviewType() is what determines if the popup is a tooltip or a full popover.  The invaildType in the expression.js is what avoids showing the popup when over a number, string, or atom class